templates/ai/contact-us.html

```diff
@@ -0,0 +1,33 @@
+{% extends "ai/base_ai.html" %}
+{% block title %}Contact us | Artificial Intellegence {% endblock %}
+
+
+{% block content %}
+  {% if product == 'ai-index-workshop' %}
+    {% include "shared/_cloud-contact-us-form.html" with  h1="Contact us about our AI workshop" intro_text="Fill in your details below and a member of our sales team will be in touch." formid='3231'  lpId='6279' returnURL='https://www.ubuntu.com/ai/thank-you?product=ai-index-workshop' %}
+  {% elif product == 'ai-index-assessment' %}
+    {% include "shared/_cloud-contact-us-form.html" with  h1="Contact us about our ML / Data Science assessment" intro_text="Fill in your details below and a member of our sales team will be in touch." formid='3231'  lpId='6279' returnURL='https://www.ubuntu.com/ai/thank-you?product=ai-managed' %}
+  {% elif product == 'ai-index-managed' %}
+    {% include "shared/_cloud-contact-us-form.html" with  h1="Contact us about our managed AI services" intro_text="Fill in your details below and a member of our sales team will be in touch." formid='3231'  lpId='6279' returnURL='https://www.ubuntu.com/ai/thank-you?product=ai-managed' %}
+  {% elif product == 'ai-install' %}
+    {% include "shared/_cloud-contact-us-form.html" with  h1="Contact us about installing Kubeflow" intro_text="Fill in your details below and a member of our sales team will be in touch." formid='3231'  lpId='6279' returnURL='https://www.ubuntu.com/ai/thank-you?product=ai-managed' %}
+  {% elif product == 'ai-consulting' %}
+    {% include "shared/_cloud-contact-us-form.html" with  h1="Contact us about Canonical consulting" intro_text="Fill in your details below and a member of our sales team will be in touch." formid='3231'  lpId='6279' returnURL='https://www.ubuntu.com/ai/thank-you?product=ai-managed' %}
+  {% else %}
+    {% include "shared/_cloud-contact-us-form.html" with  h1="Contact us about AI on Canonical" intro_text="Fill in your details below and a member of our sales team will be in touch." formid='3231'  lpId='6279' returnURL='https://www.ubuntu.com/ai/thank-you?product=ai-managed' %}
```

Comment:
This one should be for `ai-features` and the default should be `Contact us about Artificial Intelligence`.